prov/util/src/util_mr_map.c

```diff
@@ -52,7 +52,6 @@ dup_mr_attr(const struct fi_mr_attr *attr, uint64_t flags)
 	dup_attr->mr_iov = (struct iovec *) (dup_attr + 1);
 
 	/*
-	 * dup_mr_attr is only used insided ofi_mr_map_insert.
```

Comment:
sorry one last comment - this was changed from static and declared in the header but I don't see it called in this PR?

Reply:
probably leftover crud from a prior attempt. will revert